src/module/data/SR5ItemDataWrapper.ts

```diff
@@ -274,6 +274,10 @@ export class SR5ItemDataWrapper extends DataWrapper<ShadowrunItemData> {
         return { fire: fire ?? 0, electricity: electricity ?? 0, cold: cold ?? 0, acid: acid ?? 0, radiation: radiation ?? 0 };
     }
 
+    getLinkedActorUuid(): string | undefined {
+        return this.getData().linkedActor;
```

Comment:
This introduces a data property to the contact items data schema.

You've added it to `src\module\types\item\Contact.ts` but it should also be added to `template.json` to have clean data on new and existing contacts, without creating new data when linking an actor while this not exist until then. That way we can have a clean default value ('' or null) before and after an actor has been linked to a contact.
Let me know, if you need help with `template.json`.

Reply:
I just hope that this is correct in the template.json